packages/client/src/views/RegistrationHome/RowHistoryView.tsx

```diff
@@ -332,7 +332,16 @@ export class RowHistoryViewComponent extends React.Component<IProps> {
   getRenderedData() {
     const { intl } = this.props
     const transformedData = this.transformer()
-
+    if (!window.config.EXTERNAL_VALIDATION_WORKQUEUE) {
+      transformedData.operationHistories.forEach(item => {
+        if (item.type === 'WAITING_VALIDATION') {
+          transformedData.operationHistories.splice(
```

Comment:
We can just filter histories with `WAITING_VALIDATION` status out while rendering it